src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -92,15 +92,15 @@
  *   -weights path/to/my_weights.hd5
  * </pre>
  */
+@BetaFeature
```

Comment:
I've done only superficial review of the python code behind this. Has anyone else ever reviewed it ? We should talk about how to get that done.

Reply:
@mbabadi did a review too, but I would be happy for someone to take closer look if there's a volunteer.